src/components/basic/Badge.tsx

```diff
@@ -62,14 +53,19 @@ const Badge = forwardRef<HTMLDivElement, BadgeProps>(function BadgeFn(
 			<Comp
 				ref={ref}
 				orientation={'column'}
-				padding={{ vertical: '4px', horizontal: '8px' }}
-				height={isBadgeCircle ? '1rem' : '20px'}
+				padding={isBadgeCircle ? undefined : { vertical: 'extrasmall', horizontal: 'small' }}
+				height={isBadgeCircle ? '1rem' : '1.25rem'}
 				width={isBadgeCircle ? '1rem' : 'auto'}
-				background={getColor(backgroundColor, theme)}
-				$color={color}
+				background={backgroundColor}
 				{...rest}
 			>
-				{icon ? <Icon icon={icon} color={color} /> : badgeText}
+				{icon ? (
+					<Icon icon={icon} size={'small'} color={color} />
+				) : (
+					<Text color={color} size={'extrasmall'} weight={'regular'} lineHeight={1.125}>
```

Comment:
line-height must be 1.5 (you can see in figma 150%, or 1.125rem, which is 0.75rem * 1.5)